src/main/java/com/kilopo/kosshop/entity/Category.java

```diff
@@ -0,0 +1,48 @@
+package com.kilopo.kosshop.entity;
+
+import javax.persistence.*;
```

Comment:
change wildcard import to standard imports (put setting in idea)

and format code (Ctrl + Alt + L in Idea)